docs/api/embed.rst

```diff
@@ -0,0 +1,135 @@
+Embed API
+=========
+
+Read the Docs allows you to embed content from any of the projects we host.
+This allows reuse of content across sites, making sure the content is always up to date.
+
+.. contents::
+   :local:
+
+Usage
+-----
+
+Check our :doc:`/guides/embedding-content` guide to learn how to use the embed API in your projects.
+You can also use the API with our `sphinx-hoverxref`_ Sphinx extension.
+
+.. _sphinx-hoverxref: https://sphinx-hoverxref.readthedocs.io
+
+API
+---
+
+The embed API is exposed from the domain where your docs are being served.
+This is ``https://docs.readthedocs.io/_/api/v2/embed`` for the ``docs`` project, for example.
+
+.. warning::
+
+   This API isn't stable yet, feel free to `open an issue`_ if you find any problems.
+   Only Sphinx projects are supported at the moment,
+   a new version of this API with more general support is on the way!
+
+   .. _open an issue: https://github.com/readthedocs/readthedocs.org/issues/new
+
+.. http:get:: /_/api/v2/embed/
+
+   Return the content of a section with all links rewritten to be absolute.
+   There are two ways of using the API
+
+   Using the project and version slug.
+
+   .. Request
+
+   :query project: Project slug
+   :query version: Version slug
+   :query path:
+      Path to the HTML page relative the the root of your documentation,
+      this is without the language or version.
+   :query section:
+      ID from the HTML element,
+      this is usually the anchor part used in a URL to link to a section.
```

Comment:
So, a tangent on silly english writing rules: the a/an rule is actually particularly confusing and fairly subjective even, because which article you use is based on the initial sound of the following word:
https://www.merriam-webster.com/words-at-play/is-it-a-or-an

I would say "a URL" is correct, "an URL" doesn't sound correct because the `U` provides a consonant sound, not a vowel sound. This can get confusing really quick due to first/native languages, dialects, and language evolution.

Reply:
(╯°□°）╯︵ ┻━┻